src/ZoweUSSNode.ts

```diff
@@ -101,7 +101,10 @@ export class ZoweUSSNode extends ZoweTreeNode implements IZoweUSSTreeNode {
             this.setProfile(Profiles.getInstance().loadNamedProfile(mParent.mProfileName));
         }
         this.etag = etag ? etag : "";
-        utils.applyIcons(this);
+        const icon = getIconByNode(this);
```

Comment:
No need to specify the `index.ts` file when importing things :smile:
`import { getIconByNode } from "./generators/icons/index";`
I know it could've been an auto-import functionality from VSCode, so no worries : )

No need to change it here since most imports will change based on the cyclic dependency issue.

No changes requested!